### PR TITLE
Fix issue preventing copying text out of theme storybook documentation

### LINKED ISSUE #23998: [Bug]: Unable to copy colors from the v9 documentation
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
All browsers and OS
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://react.fluentui.dev/?path=/docs/theme-color--page

### Bug Description

In the Color section of the v9 documentation: https://react.fluentui.dev/?path=/docs/theme-color--page

When I press key `Cmd/Ctrl + C` to copy a selected color token or HEX color, it will show "Copied" indicator on the bottom of the section (next to 'Show code'), but it will not copy the selected text of the color, hence unable to paste.

The same goes when I right-click on the color and Copy the selected text - it will not be copied.

Repro steps:
1. Go to https://react.fluentui.dev/?path=/docs/theme-color--page
2. Select text from the color palette (HEX or token)
3. Right-click -> copy or Cmd/Ctrl + C

## Actual Behavior
The text is not copied and paste does not work or paste previous copied things.

## Expected Behavior
Paste (Cmd/Ctrl + V) will paste the text

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #24130: [Bug]: Cannot copy token names from doc site
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
- Windows 11
  - Chrome
  - Edge
  - Firefox
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://react.fluentui.dev/?path=/docs/theme-color--page

## Actual Behavior

1. Select a token name (e.g., `colorBrandBackground2`).
2. Copy it (e.g., `ctrl + c`, right click > copy)
3. Paste it somewhere else (e.g., `ctrl + v`, right click > paste)
4. Token was not copied to clipboard

copy_paste

## Expected Behavior

I should be able to copy and paste token names from the doc site.

### Logs

_No response_

### Requested priority

High

### Products/sites affected

react.fluentui.dev

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by behowell**
## Current Behavior

It is not possible to copy text out of theme documentation on these pages:
* https://react.fluentui.dev/?path=/docs/theme-color--page
* https://react.fluentui.dev/?path=/docs/theme-motion-and-size--page
* https://react.fluentui.dev/?path=/docs/theme-typography--page

The root cause is a bug in storybook:
* https://github.com/storybookjs/storybook/issues/18611

## New Behavior

A workaround for the storybook bug is to turn off the "Show code" buttons for these stories:
image

In this case, there is nothing interesting in the code (it just shows a wrapper component like `<Colors />`), so there's no harm in removing it for the theme documentation. In fact, for that reason it's probably clearer to _not_ have the "Show code" button, so this change should remain even after the underlying storybook issue is fixed.

## Related Issue(s)

* Fixes #23998
* Fixes #24130

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `78.972 kB`<br />`24.012 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `82.123 kB`<br />`20.499 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |   `48.533 kB`<br />`13.78 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          | `138.314 kB`<br />`40.998 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `68.248 kB`<br />`19.109 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `22.494 kB`<br />`7.157 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `23.397 kB`<br />`7.449 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `23.947 kB`<br />`7.022 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `35.716 kB`<br />`9.503 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `42.789 kB`<br />`10.738 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `38.334 kB`<br />`10.368 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `45.667 kB`<br />`11.715 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `50.941 kB`<br />`10.924 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `67.014 kB`<br />`19.206 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `62.696 kB`<br />`18.126 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `8.461 kB`<br />`3.555 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `9.504 kB`<br />`3.896 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |     `8.562 kB`<br />`3.61 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `74.425 kB`<br />`23.999 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `73.938 kB`<br />`23.991 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `190.54 kB`<br />`52.487 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.036 kB`<br />`10.577 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `83.707 kB`<br />`25.013 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `16.321 kB`<br />`5.837 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |    `10.68 kB`<br />`4.215 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `23.554 kB`<br />`7.644 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `9.238 kB`<br />`3.815 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |    `12.082 kB`<br />`4.88 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `117.801 kB`<br />`35.808 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             |     `121 kB`<br />`36.288 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `10.839 kB`<br />`4.146 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `105.483 kB`<br />`32.077 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |    `9.844 kB`<br />`3.677 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |   `23.854 kB`<br />`8.299 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `14.913 kB`<br />`5.658 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |  `30.481 kB`<br />`10.406 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |   `14.319 kB`<br />`5.711 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `20.746 kB`<br />`7.299 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `26.136 kB`<br />`8.387 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `44.227 kB`<br />`12.492 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `19.932 kB`<br />`6.363 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `26.951 kB`<br />`8.688 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `11.572 kB`<br />`4.537 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `14.882 kB`<br />`4.977 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `23.078 kB`<br />`7.707 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |   `28.995 kB`<br />`6.215 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |    `16.973 kB`<br />`4.86 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `44.853 kB`<br />`15.406 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c69f9bd19610cbceb4ecf8243b5d49afea4634d7'>c69f9bd19610cbceb4ecf8243b5d49afea4634d7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/Avatar-mount_base.html">1510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/Avatar-mount.html">1510</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/Button-mount_base.html">1053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/Button-mount.html">1058</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProvider-mount_base.html">1842</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProvider-mount.html">1818</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProviderWithTheme-mount_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProviderWithTheme-mount.html">717</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">673</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">721</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">711</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/MakeStyles-mount_base.html">2197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/MakeStyles-mount.html">2220</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/SpinButton-mount_base.html">2995</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24135/perf-test-react-components/SpinButton-mount.html">2996</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-theme-fbb9a9be-f6c1-44e6-8090-06571347651a.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Fix issue preventing copying text out of theme storybook documentation",
```
Comment (khmakoto) [inline change/@fluentui-react-theme-fbb9a9be-f6c1-44e6-8090-06571347651a.json:3]:
```suggestion
  "comment": "fix: Fix issue preventing copying text out of theme storybook documentation.",
```

**Review by khmakoto — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

File: change/@fluentui-react-theme-fbb9a9be-f6c1-44e6-8090-06571347651a.json (hunk 1 above)
Comment (behowell) [reply change/@fluentui-react-theme-fbb9a9be-f6c1-44e6-8090-06571347651a.json]: Ok, I fix fixed the description 👍

**Pull request merged**